docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -102,11 +106,148 @@ \section{Critical Assumptions}
 		& \multicolumn{1}{|c|}{\textbf{Recommended Action}}
 		& \multicolumn{1}{|c|}{\textbf{SR}}
 		& \multicolumn{1}{|c|}{\textbf{Ref.}}  \\
-		\hline
+        \hline
+        Authenti\-cation
+        & User is unable to login
+        & User cannot access their account and application features
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item User entered incorrect credentials
+        \end{enumerate}
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item Allow users to reset their password
+        \end{enumerate}
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item SR1
+        \end{enumerate}
+        & HA1 \\
+        \cline{2-7}
+        & User's account is hacked by a third-party
+        & User's data can be read and modified by third-party
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item Third-party gains access to the  user's account without their permission
+        \end{enumerate}
+        & \begin{enumerate}[label=\alph*., leftmargin=*]
+            \item Refer to HA1-a
+            \item Limit the number of login attempts a person has for a given time period
```

Comment:
should we put a number of login attempts and define how long the wait is?

Reply:
Maybe define it in Section 6 as part of your description/rationale/fit criterion for that requirement @Tanger71